Networking/Networking/Mapper/OrderRefundsMapper.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+
+/// Mapper: OrderRefund
+///
+struct OrderRefundsMapper: Mapper {
+
+    /// Site Identifier associated to the order refund that will be parsed.
+    ///
+    /// We're injecting this field via `JSONDecoder.userInfo` because SiteID is not returned in any of the Order Endpoints.
+    ///
+    let siteID: Int
+
+
+    /// (Attempts) to convert a dictionary into [OrderRefund].
+    ///
+    func map(response: Data) throws -> [OrderRefund] {
+        let decoder = JSONDecoder()
+        decoder.dateDecodingStrategy = .formatted(DateFormatter.Defaults.dateTimeFormatter)
+        decoder.userInfo = [
+            .siteID: siteID
+        ]
+
+        return try decoder.decode(OrderRefundsEnvelope.self, from: response).orderRefunds
+    }
+}
+
+
+/// OrdersRefundsEnvelope Disposable Entity
+///
+/// `Load Order` endpoint returns the requested order document in the `data` key. This entity
```

Comment:
Is the endpoint mentioned in the comment accurate? It seems to me like `Load Order refunds` might be more accurate.

Reply:
You are right, fixed.